app/templates/orders/detail.hbs

```diff
@@ -51,7 +51,11 @@
           Organisation:
         </div>
         <div class="small-8 columns">
-          {{apply-line-break model.organisation.name}}
```

Comment:
I think we can use inline **if** condition here.
What do you think @viveksriram1 ?

Reply:
I tried inline if condition but its not showing properly because of apply-line-break